lib/galaxy/datatypes/mothur.py

```diff
@@ -582,6 +582,7 @@ def sniff(self, filename):
 
 class Frequency(Tabular):
     file_ext = 'mothur.freq'
+    max_snif_lines = 25
```

Comment:
You don't need to re-implement this, there's already a `count` parameter for `iter_headers()`, which by default is 60 lines.